### DISCUSSION TITLE
What happens with the wallpaper from youtube's url

### DISCUSSION THREAD

**Discussion by shinobitiger310** [rocksdanister/lively#326] [Q&A] [answered]
I have a concern about the wallpaper from youtube's url. Will Livelywallpaper download the video every time we use that wallpaper from youtube? Sorry for my bad English.

**Comment by rocksdanister** [comment, marked as answer]
Yes, its streaming using mpv player.

**Reply by shinobitiger310** [reply]
Thanks for your reply. In that case, maybe it will take a lot of internet data. I think my solution is I will export it as zip file and add new wallpaper using that zip file.

**Reply by rocksdanister** [reply]
zip will just store the url for video stream.
You will have to use some third party software and download the video file itself.
